codec-http2/src/test/java/io/netty/handler/codec/http2/HpackDecoderTest.java

```diff
@@ -52,6 +57,9 @@
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 
 public class HpackDecoderTest {
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```

Comment:
fyi, in the future, this should be final

Reply:
https://github.com/netty/netty/pull/7596